src/pages/index.js

```diff
@@ -1,90 +1,233 @@
+    .then((userData) => {
+      avatarFormElement.reset();
+      profileImagePopupForm.close();
+      userInfo.setUserImage(userData.avatar);
+    })
+    .catch(console.error)
+    .finally(() => profileImagePopupForm.setLoading(false));
+}
+
+// Initialize Add Card Popup Form
+const cardPopupForm = new PopupWithForm(
+  "#add-card-modal",
+  handleNewCardFormSubmit
 );
-editProfilePopup.setEventListeners();
+cardPopupForm.setEventListeners();
+
+// Add New Card Form Submit Handler
+function handleNewCardFormSubmit(userInfo) {
+  cardPopupForm.setLoading(true);
+  api
+    .fetchNewCard(userInfo)
+    .then((res) => {
+      cardList.addItem(res);
+    })
+    .then(() => {
+      addCardFormElement.reset();
+      addFormValidator.disableButton();
+      cardPopupForm.close();
+    })
+    .catch(console.error)
+    .finally(() => cardPopupForm.setLoading(false));
+}
+
+// Initialize Popup With Image
+const popupWithImage = new PopupWithImage("#preview-picture-modal");
+popupWithImage.setEventListeners();
 
-// Initialize UserInfo
-const userInfo = new UserInfo(profileSelectors);
+// Handle Preview Picture
+function handlePreviewPicture(cardData) {
+  popupWithImage.open(cardData);
+}
 
-document.querySelector("#profile-edit-button").addEventListener("click", () => {
+// Initialize Profile Popup Form
+const profilePopupForm = new PopupWithForm(
+  "#profileEditModal",
+  handleProfileEditSubmit
+);
+profilePopupForm.setEventListeners();
+
+// Profile Edit Button Click Event
+profileEditButton.addEventListener("click", () => {
+  profilePopupForm.open();
   const userData = userInfo.getUserInfo();
-  editProfilePopup.setInputValues({
-    title: userData.name,
-    description: userData.job,
-  });
-  editProfilePopup.open();
+  profileTitleInput.value = userData.title;
+  profileDescriptionInput.value = userData.description;
+  editFormValidator.disableButton();
 });
 
-document.querySelector("#modal-add-button").addEventListener("click", () => {
-  addCardPopup.open();
-});
+// Add New Card Button Click Event
+addNewCardButton.addEventListener("click", () => cardPopupForm.open());
+
+// Initialize Delete Confirmation Popup
+const deleteConfirmPopup = new PopupWithConfirmation("#confirm-modal");
+deleteConfirmPopup.setEventListeners();
+
+// Handle Delete Click
+function handleDeleteClick(cardElement) {
+  deleteConfirmPopup.open();
+
+  deleteConfirmPopup.setSubmitAction(() => {
+    deleteConfirmPopup.setLoading(true);
+    api
+      .fetchDeleteCard(cardElement.getId())
+      .then(() => {
+        cardElement.removeCard();
+        deleteConfirmPopup.close();
+      })
+      .catch(console.error)
+      .finally(() => deleteConfirmPopup.setLoading(false));
+  });
+}
+
+// Handle Like Click
+function handleLikeClick(cardElement) {
+  if (cardElement.getLikes()) {
+    api
+      .fetchDisLikeCard(cardElement.getId())
+      .then((res) => {
+        cardElement.renderLikes(res.isLiked);
+        cardElement._isLiked = res.isLiked;
```

Comment:
`_isLiked`

Please, delete the **low dash** (`underscore`) from methods if you want to use them `outside` a class because the `low dash` shows a **private** method which is used only inside the class.